2024/m365ahmedabadusergroup/data.json

```diff
@@ -0,0 +1,25 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "Kirti Prajapati"
```

Comment:
I think you are missing a dash between your first and last name. This should be your GitHub handle and not your name.

Reply:
@kirti-prajapati , can you please address the inline comment?

Reply:
Hi @tiagocostapt updated the file.